scene/main/node.h

```diff
@@ -120,25 +117,47 @@ class Node : public Object {
 		Map<StringName, MultiplayerAPI::RPCMode> rpc_methods;
 		Map<StringName, MultiplayerAPI::RPCMode> rpc_properties;
 
+		int process_priority;
+
 		// variables used to properly sort the node when processing, ignored otherwise
 		//should move all the stuff below to bits
```

Comment:
It's to group the bools together. :slightly_smiling_face: 
As they are now 1 bit each, it's important to define them together to get the benefit so they are stored in one continuous `uint`. If the `process_priority` was in between it would force them into different bytes afaik. So what was once probably about 64 bytes is now probably in about 4.